Analysis/Tasks/PWGHF/qaTaskSimple.cxx

```diff
@@ -155,28 +155,59 @@ struct QAGlobalObservables {
 
 /// Task to QA the kinematic properties of the tracks
 struct QATrackingKine {
-  o2fw::Configurable<int> nBinsPt{"nBinsPt", 100, "Number of bins for Pt"};
-  std::array<double, 2> ptRange = {0, 10.};
-  std::array<double, 2> invptRange = {-1.e2, 1.e2};
-  o2fw::Configurable<int> nBinsEta{"nBinsEta", 100, "Number of bins for the eta histogram."};
-  std::array<double, 2> etaRange = {-6, 6};
-  o2fw::Configurable<int> nBinsPhi{"nBinsPhi", 100, "Number of bins for Phi"};
+  o2fw::Configurable<int> pdgCode{"pdgCode", 0, "PDG code of the particle to select in absolute value, 0 selects every particle"};
+
+  o2fw::Configurable<float> etaMin{"eta-min", -3.f, "Lower limit in eta"};
+  o2fw::Configurable<float> etaMax{"eta-max", 3.f, "Upper limit in eta"};
+  o2fw::Configurable<float> phiMin{"phi-min", 0.f, "Lower limit in phi"};
+  o2fw::Configurable<float> phiMax{"phi-max", 2.f * M_PI, "Upper limit in phi"};
+  o2fw::Configurable<float> ptMin{"pt-min", 0.f, "Lower limit in pT"};
+  o2fw::Configurable<float> ptMax{"pt-max", 5.f, "Upper limit in pT"};
+  o2fw::Configurable<int> ptBins{"pt-bins", 500, "Number of pT bins"};
+  o2fw::Configurable<int> etaBins{"eta-bins", 500, "Number of eta bins"};
+  o2fw::Configurable<int> phiBins{"phi-bins", 500, "Number of phi bins"};
 
   o2fw::HistogramRegistry histos{"HistogramsKineQA"};
   void init(o2fw::InitContext&)
   {
-    histos.add("tracking/pt", ";#it{p}_{T} [GeV/c]", o2fw::kTH1D, {{nBinsPt, ptRange[0], ptRange[1]}});
-    histos.add("tracking/invpt", ";1/#it{p}_{T} [GeV/c]^{-1}", o2fw::kTH1D, {{nBinsPt, invptRange[0], invptRange[1]}});
-    histos.add("tracking/eta", ";#eta", o2fw::kTH1D, {{nBinsEta, etaRange[0], etaRange[1]}});
-    histos.add("tracking/phi", ";#varphi [rad]", o2fw::kTH1D, {{nBinsPhi, 0, 2 * M_PI}});
+    o2fw::AxisSpec ptAxis{ptBins.value, ptMin.value, ptMax.value};
+    o2fw::AxisSpec etaAxis{etaBins.value, etaMin.value, etaMax.value};
+    o2fw::AxisSpec phiAxis{phiBins.value, phiMin.value, phiMax.value};
+
+    TString commonTitle = "";
+    if (pdgCode.value != 0) {
+      commonTitle += Form("PDG %i", pdgCode.value);
+    }
+
+    histos.add("tracking/pt", commonTitle + ";#it{p}_{T} [GeV/#it{c}]", o2fw::kTH1D, {ptAxis});
+    histos.add("tracking/eta", commonTitle + ";#eta", o2fw::kTH1D, {etaAxis});
+    histos.add("tracking/phi", commonTitle + ";#varphi [rad]", o2fw::kTH1D, {phiAxis});
+    histos.add("particle/pt", commonTitle + ";#it{p}_{T} [GeV/#it{c}]", o2fw::kTH1D, {ptAxis});
+    histos.add("particle/eta", commonTitle + ";#eta", o2fw::kTH1D, {etaAxis});
+    histos.add("particle/phi", commonTitle + ";#varphi [rad]", o2fw::kTH1D, {phiAxis});
```

Comment:
Since you already have `#it{c}` and `#it{p}`, you can also have `#it{#eta}` and `#it{#varphi}`. :-)

Reply:
Fixed